data-pipeline/src/data_pipeline/pipelines/genes.py

```diff
@@ -309,13 +356,15 @@ def annotate_with_preferred_transcript(table_path):
 
 pipeline.set_outputs(
     {
-        "genes_grch37": "annotate_grch37_genes_step_4",
-        "genes_grch38": "annotate_grch38_genes_step_3",
+        "genes_grch37": "annotate_grch37_genes_step_5",
+        "genes_grch38": "annotate_grch38_genes_step_4",
         "base_transcripts_grch37": "extract_grch37_transcripts",
         "base_transcripts_grch38": "extract_grch38_transcripts",
         "transcripts_grch37": "annotate_grch37_transcripts",
         "transcripts_grch38": "extract_grch38_transcripts",
         "mane_select_transcripts": "import_mane_select_transcripts",
+        "heterozygous_variant_cooccurrence_counts": "prepare_heterozygous_variant_cooccurrence_counts",
+        "homozygous_variant_cooccurrence_counts": "prepare_homozygous_variant_cooccurrence_counts",
```

Comment:
Are these anticipated to be referenced elsewhere? If they're only going to be used to annotate the final `genes` hailtables, I believe these aren't needed as pipeline outputs.

Setting pipeline outputs is used so that `export_to_elasticsearch` can find the location of the hailtables to export.

Reply:
Confirmed, don't need to set these.